emojirades/bot.py

```diff
@@ -106,7 +106,23 @@ def handle_event(self, **payload):
                 else:
                     channel = self.decode_channel(event["channel"])
 
-                webclient.chat_postMessage(channel=channel, text=response)
+                if isinstance(response, str):
+                    # Plain strings are assumed as 'chat_postMessage'
+                    webclient.chat_postMessage(channel=channel, text=response)
+                    continue
+
+                func = getattr(webclient, response["func"], None)
```

Comment:
fancy!!!

Reply:
Also thinking if it'll be easier to understand if it's an if/else? Took me a bit to realise that the rest of the code is all related to handling complex type.